src/gdb.rs

```diff
@@ -37,6 +39,7 @@ pub struct CrashContext {
     pub registers: Registers,
     pub mappings: MappedFiles,
     pub machine: MachineInfo,
+    pub pc_memory: MemoryObject,
 }
 
 impl CrashContext {
```

Comment:
impl Severity for  CrashContext. Rename CrashContext to GdbCrashContext

Reply:
done